Generals/Code/GameEngine/Source/Common/System/GameMemory.cpp

```diff
@@ -3245,11 +3245,11 @@ void MemoryPoolFactory::debugMemoryReport(Int flags, Int startCheckpoint, Int en
 	to ensure that's the case)
 */
 #if defined(_DEBUG)
-	#pragma comment(lib, "GameEngineDebug")
+	#pragma comment(lib, "GameEngineGDebug")
 #elif defined(_INTERNAL)
-	#pragma comment(lib, "GameEngineInternal")
+	#pragma comment(lib, "GameEngineGInternal")
 #else
-	#pragma comment(lib, "GameEngine")
+	#pragma comment(lib, "GameEngineG")
```

Comment:
How about we build targets of Generals & Tools and Zero Hour & Tools into 2 different folders? Is this possible?

Reply:
This has been implemented, now only runtime targets get moved and they get put under either the `Generals` or `GeneralsMD` subfolders in the build directory now.